datadog_checks_dev/datadog_checks/dev/tooling/commands/run.py

```diff
@@ -14,7 +14,7 @@
 @click.pass_context
 def run(ctx, args):
     """Run commands in the proper repo."""
-    if not args or '-h' in args or '--help' in args:
+    if not args or (len(args) == 1 and args[0] in ('-h', '--help')):
```

Comment:
IMO we should revert to Click's default behavior of following the Unix convention (see [Options-like arguments](https://click.palletsprojects.com/en/7.x/arguments/#option-like-arguments)), i.e. revert to…

```suggestion
    if not args:
```

As well as remove `context_settings=UNKNOWN_OPTIONS`.

Then usage would be…

```bash
$ ddev run <command> -- <args> <--options>
# Example:
$ ddev run tokei -- -s Files
```

Reply:
Agree using `--` seems much cleaner than trying to compensate the behaviour.

Reply:
For this case, that seems overly complicated -- really this is just a simple passthrough command to run something in the appropriate folder, it is more natural to have all the args as you would normally write them.

Reply:
I mean it's a fair point, since…

```
docker exec -it <container_id> <command with any arguments not requiring '--'>
```

Considering `ddev run` has no options other than `--help` right now keeping things simple is probably best… Resolving.